python-checks/src/main/java/org/sonar/python/checks/django/DjangoReceiverDecoratorCheck.java

```diff
@@ -22,15 +22,18 @@
 import java.util.Optional;
 import org.sonar.check.Rule;
 import org.sonar.plugins.python.api.PythonSubscriptionCheck;
+import org.sonar.plugins.python.api.quickfix.PythonQuickFix;
 import org.sonar.plugins.python.api.tree.Decorator;
 import org.sonar.plugins.python.api.tree.FunctionDef;
 import org.sonar.plugins.python.api.tree.Tree;
+import org.sonar.python.quickfix.TextEditUtils;
 import org.sonar.python.tree.TreeUtils;
 
 @Rule(key = "S6552")
 public class DjangoReceiverDecoratorCheck extends PythonSubscriptionCheck {
 
   private static final String MESSAGE = "Move this '@receiver' decorator to the top of the other decorators.";
+  private static final String QUICK_FIX_MESSAGE = "Move this '@receiver' decorator to the top of the other decorators.";
```

Comment:
I'd suggest `Move the '@receiver' decorator to the top` for the QF message, to avoid the demonstrative pronoun and be a bit more concise.

Reply:
done